themes-default/slim/static/js/store.js

```diff
@@ -171,10 +175,34 @@ const store = new Puex({
         },
         // Default handler called for all websocket methods
         [SOCKET_ONMESSAGE](state, message) {
+            const { data, event } = message;
+            const { body, hash, type, title } = data;
+
+            // Set the current message
             state.socket.message = message;
+
+            // Show the notification to the user
+            if (event === 'notification') {
+                displayNotification(type, title, body, hash);
```

Comment:
Something is weird. Using the following action, I can't seem to get more than 3 notifications to show.
Maybe it's unrelated.
`for (let i=0; i < 10; i++) { window.store.dispatch('testNotifications') }`
```js
        testNotifications(store) {
            const rand = Math.floor((Math.random() * 10000)).toString();
            store.commit(SOCKET_ONMESSAGE, {
                event: 'notification',
                data: {
                    type: `error`,
                    title: `test-${rand}`,
                    body: `test-${rand}<br><i class="test-class">hello <b>world</b></i><ul><li>item 1</li><li>item 2</li></ul>`,
                    hash: `notification-${rand}`
                }
            });
        }
```

Reply:
Are all the commits showing in the vuex dev panel?

Reply:
Yeah, I think it's unrelated. Even with `new Notification("message")` it's a bit buggy...